enterprise-suite/console_test/resources/app_service_with_only_endpoints.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: es-test-service-with-only-endpoints
+  annotations:
+    "prometheus.io/scrape": "true"
```

Comment:
Do you mind changing these to match the current tests? I think it's `console-backend-e2e.io/scrape`. This is so we can run the tests in a shared kubernetes cluster (e.g. the openshift tests).

Reply:
Updated to keep insync with bash tests